packages/components/CHANGELOG.md

```diff
@@ -8,6 +8,54 @@ menu: Changelog
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [3.0.0-pre.8](https://github.com/GetJobber/atlantis/compare/@jobber/components@2.90.0...@jobber/components@3.0.0-pre.8) (2023-02-01)
```

Comment:
Oooh how did this change show up here? I thought walter does this automatically 🤔 y dis here?

Reply:
It is from the pre-releases that we did

Reply:
do we want to publish this? I don't think we've ever published a pre release before

Reply:
If it were published as an alpha/beta maybe but as a pre probably not

Note to us: can we release in an alpha/beta/something format next time to avoid the feature branch? 😄

Reply:
I am indifferent to keeping it in the PR however looking at React's changelog they don't include the pre-release stuff in the main changelog from what I can see. I think numbered pre-releases makes more sense than alpha, beta, etc. because:
 1. Lerna supports it better
 2. Most packages I have seen use the same numbered pre release scheme